src/app/views/forgotten-password/forgottenPasswordEmailSent.jsx

```diff
@@ -1,28 +1,38 @@
 import React from "react";
-import Input from "../../components/Input";
 import PropTypes from "prop-types";
+import Panel from "../../components/panel/Panel";
 
 const propTypes = {
     validationErrors: PropTypes.shape({
         hading: PropTypes.string,
         body: PropTypes.oneOfType([PropTypes.string, PropTypes.element])
     }),
     onSubmit: PropTypes.func,
-    emailInput: PropTypes.shape({
-        id: PropTypes.string,
-        label: PropTypes.string,
-        type: PropTypes.string,
-        onChange: PropTypes.func,
-        error: PropTypes.string
-    })
+    email: PropTypes.string
 };
 
 const ForgottenPasswordEmailSent = props => {
-    // TODO this page is to be done in an upcoming task
+    const noEmailBody = (
+        <div>
+            <p>
+                If you did not get the email you can <a href={"/florence/forgotten-password"}>reset your password again.</a>
+            </p>
+            <p>
+                Alternatively, to get help and support with your Florence account, please email&nbsp;
+                <a href="mailto:publishing@ons.gov.uk">publishing@ons.gov.uk</a>
+            </p>
+        </div>
+    );
     return (
         <div className="grid grid--justify-center">
-            <div className="grid__col-3">
+            <div className="grid__col-4">
                 <h1>We sent you an email</h1>
+                <p className={"font-size--18 margin-bottom--1"}>
+                    If you have a Florence account, we’ve sent an email to <b>{props.email}</b>.
+                </p>
+                <p>You need to follow the link in the email to reset your password.</p>
```

Comment:
Do we need to add margin-bottom--1 or was the gap too big and hence ignored?

Reply:
Current:
![Screenshot 2021-08-10 at 08 31 57](https://user-images.githubusercontent.com/47502916/128827416-764ce4cb-f87d-45af-a391-eee52a9b47d0.png)
With Margin
![Screenshot 2021-08-10 at 08 35 41](https://user-images.githubusercontent.com/47502916/128827417-ddb6c6b5-a44c-47a2-a3f9-8e0a21849a3a.png)
Initial design
![Screenshot 2021-08-10 at 08 36 34](https://user-images.githubusercontent.com/47502916/128827421-83847b90-0641-468f-8565-f045ab941372.png)

Ideally, we should have smaller margin thresholds in Florence, but by keeping to this pattern we can do either of the top two images. I initially opted for the first of the two images, but so long as the information is on the page and clear I honestly don't mind which as I don't think it makes much difference to the user. I will toggle it over to have a larger margin for now.